src/db/_check-port.js

```diff
@@ -2,7 +2,7 @@ let net = require('net')
 module.exports = function checkPort(port, fn) {
   let tester = net.createServer()
   .once('error', function (err) {
-    if (err.code != 'EADDRINUSE') return fn(err)
+    if (err.code == 'EADDRINUSE') return fn(err)
```

Comment:
This works, but I think we should probably just change this to be `if (err) return fn(err)` – will do post-merge. Thanks!